app/data.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace GitHubToMysql;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Exception\UniqueConstraintViolationException;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+
+class data {
```

Comment:
Let's follow a few principles here: 
- uppercase names for classes
- separation of concerns
- useful constructors

I suggest splitting this class in 2, one for database write operations, and one for http fetch operations.